package.json

```diff
@@ -67,6 +67,7 @@
     "test:integration": "./test/integration/scripts/start.sh && npm run test:int",
     "test:functional": "true",
     "regenerate": "yo swaggerize:test --framework hapi --apiPath './src/interface/swagger.json'",
+    "package-lock": "docker run --rm -it quoting-service:local cat package-lock.json > package-lock.json",
```

Comment:
What is the purpose of this?

Reply:
No idea. This looks like a bad merge on my part of minor/iso but idk where this came from since it's not in minor/iso itself. Removed.